### ISSUE TITLE
[Data] Label Validation Interface

### ISSUE THREAD

**Issue by charlieforward9** [charlieforward9/visualPT#49]
Admins need a way to access the dataset in a convenient manner to validate the labels on the data.

**Considerations**
- Should this be integrated into the application? Or developed as a separate process? 
- Is it possible and efficient to access Anplify data outside of the application?

**Proposal**
The safest way of validating the data would be to create a "Tinder" type interface where each labeled image is shown with its corresponding label and the user can confirm or change the label, sending the image to its respective _verified_ container.

**Comment by charlieforward9** [comment]
@DrBubbles42 I am super close to finishing this. I have the wireframe, the validation button and the logic all in place. Whats left is an error with rendering the **Image** itself, since I have to use one library to process the image for TFLITE and another for rendering it to the screen. 

TLDR; Image compatibility issue. I will have an update for you tomorrow. I apologize for the delay.

**Comment by charlieforward9** [comment]
Waiting on https://github.com/brendan-duncan/image/issues/555
